mrblib/models/pin.rb

```diff
@@ -10,16 +10,16 @@ class Pin
   def initialize(pin, inout, pullup)
```

Comment:
After using mruby/c 2.1.0, An warnings of "ArgumentError : wrong number of arguments" is emerged, because line 1 of mrblib/loops/master.rb.gpio-1 is "led1 = Pin.new(13, Pin::Out)".

Reply:
ここは居田君に任せた方が良いかもしれません．それで良ければ，このまま merge したいと思います.